docs/openapi/components/schemas/credentials/EnvironmentFootprintCredential.yml

```diff
@@ -0,0 +1,145 @@
+type: object
+properties:
+  '@context':
+    type: array
+    readOnly: true
+    const:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+    default:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+    items:
+      type: string
+      enum:
+        - https://www.w3.org/2018/credentials/v1
+        - https://w3id.org/traceability/v1
+  type:
+    type: array
+    readOnly: true
+    const:
+      - VerifiableCredential
+      - EnvironmentFootprintCredential
+    default:
+      - VerifiableCredential
+      - EnvironmentFootprintCredential
+    items:
+      type: string
+      enum:
+        - VerifiableCredential
+        - EnvironmentFootprintCredential
+  id:
+    type: string
+    format: uri
+  issuanceDate:
+    type: string
+    format: date-time
+  expirationDate:
+    type: string
+    format: date-time
+  issuer:
+    $ref: ../common/Organization.yml
+  credentialSchema:
+    type: object
+    properties:
+      id:
+        title: Id
+        description: The url of the schema file to validate the shape of the json object
+        type: string
+        format: uri
+        example: https://w3id.org/traceability/openapi/components/schemas/credentials/EnvironmentFootprintCredential.yml
+        default: https://w3id.org/traceability/openapi/components/schemas/credentials/EnvironmentFootprintCredential.yml
+        readOnly: true
+      type:
+        title: Type
+        description: The type of validation to be run against the defined schema
+        const: OpenApiSpecificationValidator2022
+  credentialSubject:
+    title: Environment Footprint
+    description: Environmental impact of a product or event.
+    type: object
+    properties:
+      impactTimeStart:
+        type: string
+        format: date-time
+        title: Impact Start time
+        description: the time when the activity impacting the environment started
+      impactTimeEnd:
+        type: string
+        format: date-time
+        title: Impact End Time
+        description: the time when the activity impacting the environment ended
+      pollutants:
+        title: Pollutants
+        description: a list of pollutants that we are tracking the environmental impact of, this is a small non-exhaustive list that we intend to expand.
+        type: object
+        properties:
+          type:
+            type: array
+            readOnly: true
+            const: 
+              - Pollutants
+            default:
+              - Pollutants
+            items:
+              type: string
+              enum:
+              - Pollutants
+          CO2:
+            title: CO2
+            description: amount of carbon dioxide released into the atmosphere, measured in kilograms
+          CH4:
+            title: CH4
+            description: amount of methane released into the atmosphere, measured in kilograms
```

Comment:
We usually have a separate unit property. Is there a reason to go with implicit (description-based) units here?

Reply:
I went off the milltestReport way of doing it: 
https://github.com/w3c-ccg/traceability-vocab/blob/78f700209d7fc0437cc66af61f9b40e7980a670d/docs/openapi/components/schemas/credentials/MillTestReportCredential.yml#L1108